src/scripts/lexmatch-sssom-compare.py

```diff
@@ -176,6 +176,8 @@ def extract_unmapped_matches(input: str, matches: TextIO, output_dir: str, summa
         ont2 = ont.upper()
         if ont == "omim":
             ont2 = "|".join((["OMIM", "OMIMPS"]))
+        elif ont == "ordo":
+            ont2 = "|".join((["ORDO", "Orphanet"]))
```

Comment:
@matentzn : this missing condition was the culprit of not catching `Orphaned:XXXX` terms. As per the incorrect prefix, my guess is someone ran the script without providing `-c metadata/mondo.sssom.config.yml` and pushed it to GitHub. I don't have any other explanation.

Reply:
Ok, thank you! Perfect detective work!